.vsts-ci/variables/build-images.yml

```diff
@@ -0,0 +1,9 @@
+variables:
+  - name: imgWin10
+    value: 'vs2017-win2016'
+  - name: imgWin7
+    value: 'vs2015-win2012r2'
+  - name: imgMacOS
+    value: 'macOS-10.14'
+  - name: imgUbuntu
+    value: 'ubuntu-16.04'
```

Comment:
The point is that we don't have a job per image. We have two jobs using a Win10 image. And we might have two more on a Win7 image.

I'm happy to take it out, but eventually an image will be deprecated and we'll have to update in multiple places, rather than associating our conceptual platform with an actual image.

Reply:
Won't we have to delete the 2 jobs that use the deprecated image anyway? Seems like we'd still be updating in multiple places regardless.

Like... what does calling it `imgWin10` get us vs `vs2017-win2016`? I guess maybe they switch to vs2019, then we update it in one place... but that's going to be so rare.

Reply:
> Like... what does calling it imgWin10 get us vs vs2017-win2016? I guess maybe they switch to vs2019, then we update it in one place... but that's going to be so rare.

Yeah that's the scenario I'm targeting. Basically to have a small file that just tracks what our CI platforms are away from the noise, to separate the conceptual platforms from whatever the images AzDO actually provides are. Because we don't think in Windows Server 2016 and macOS 10.13, we think in Win 10 vs Win 7 vs Mac vs Ubuntu.

It doesn't stop us creating jobs, it keeps the list of images we use separate from the jobs.

Anyway, I've deleted it.